src/routes/(authentication)/login/+layout.svelte

```diff
@@ -1,25 +1,36 @@
 <script lang="ts">
-  import { AppBar } from '@skeletonlabs/skeleton';
-  import logo from '$lib/assets/app-logo.png';
   import '../../../app.postcss';
   import { user } from '$lib/stores/User';
   import { goto } from '$app/navigation';
   import { onMount } from 'svelte';
+  import Footer from '$lib/components/Footer.svelte';
+  import Dots from '$lib/components/Dots.svelte';
+
   onMount(() => {
     if ($user && $user.token) {
       goto('/');
     }
   });
 </script>
 
-<header class="w-full">
-  <AppBar padding="py-3 pl-2 pr-5" background="bg-surface-50-900-token">
-    <svelte:fragment slot="lead">
-      <a href="/" aria-current="page" data-testid="logo-home-link">
-        <img id="nav-logo" alt="PIC-Sure logo" src={logo} class="mx-1" />
-      </a>
-    </svelte:fragment>
-  </AppBar>
-</header>
+<div class="w-full full-height">
+  <Dots class="top-dots" />
+  <slot />
+  <Dots class="bottom-dots" />
+  <div class="footer">
+    <Footer showSitemap={false} />
+  </div>
+</div>
 
-<slot />
+<style>
+  .full-height {
+    height: calc(100% - 56px);
+  }
+  .footer {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: #f5f5f5;
```

Comment:
Is this color something we can make a variable for at the postcss level? Might be annoying to find it later.

Reply:
This class is never used haha